src/explorer/setDefaults.js

```diff
@@ -0,0 +1,16 @@
+import defaultSettings from './defaultSettings';
+
+export function setDefaults(explorer) {
+  /********************* ignoredColumns *********************/
+  var firstKey = Object.keys(explorer.config.files[0])[0];
```

Comment:
This was a typo - Label column and ignored columns default were transposed. Was fixed in 2nd PR.